subsys/bluetooth/audio/audio.c

```diff
@@ -0,0 +1,99 @@
+/* Common functions for LE Audio services */
```

Comment:
Sorry, but as discussed, no one wants the `128B` flag anymore to be in the GATT API. Thus the service has to validate the additional security requirements on service level in GATT Read/Write callback context. The good thing in this approach s that the security is checked in one place. IMO it's advantage, rather than disadvantage.

Another approach I was thinking of was to have macro/function testing the security. This would work for `BT_GATT_CHARACTERISTIC` but for `BT_GATT_CCC` we would still need to use `BT_GATT_CCC_MANAGED` instead with `cfg_write` validating the security. This, I decided that it's cleaner to `override` the GATT API.

Reply:
The CCC check is a good reason for this approach :)